alpha/alpha-server/src/test/java/org/apache/servicecomb/saga/alpha/server/AlphaIntegrationTest.java

```diff
@@ -134,6 +150,23 @@
   private final CompensationStreamObserver compensateResponseObserver = new CompensationStreamObserver(
       this::onCompensation);
 
+  private static SslContext getSslContext(){
+    ClassLoader classLoader = AlphaIntegrationTest.class.getClassLoader();
+    SslContext sslContext = null;
+    try {
+      sslContext = GrpcSslContexts.forClient().sslProvider(SslProvider.OPENSSL)
+          .protocols("TLSv1.2","TLSv1.1")
+          .ciphers(Arrays.asList("ECDHE-RSA-AES128-GCM-SHA256",
+              "ECDHE-RSA-AES256-GCM-SHA384",
+              "ECDHE-ECDSA-AES128-SHA256"))
+          .trustManager(new File(classLoader.getResource("ca.crt").getFile()))
```

Comment:
I think we need to use the environment property instead of hard code here.

Reply:
@WillemJiang  How about we put this configuration under the resources folder. Then we can update the protocol/ciphers without modifying the code and it's also not exposed to end-user.

Reply:
Fixed. The ciphers and protocols are now read from resources/ssl.properties